ui/.storybook/main.ts

```diff
@@ -23,6 +23,7 @@ const config: StorybookConfig = {
   },
   docs: {
     autodocs: 'tag',
+    docsMode: true,
```

Comment:
This removes the list of individual stories from the Storybook directory. They were not necessary, as each story is rendered inside their Docs page